surveys/2023-04/questions.md

```diff
@@ -26,6 +26,8 @@
     - Library & package authors: Users who write libraries and packages to be included on other applications
     - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
     - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+    - Education
```

Comment:
Education is part of the previous line, should we have it duplicated or remove one of them ?

Reply:
I think we should remove "education" in the brackets.

Reply:
It's possible that we decided to just include those groups in the "Organization with investments in Node.js" in which case we could leave as is. @sheplu, @AugustinMauroy what do you think?

Reply:
I was thinking about it this week end, and having a dedicated part for eduction seems good. But in that case should we also move the `non-profit` to a specific one ?